casanovo/denovo/evaluate.py

```diff
@@ -310,3 +315,130 @@ def aa_precision_recall(
     n_aa_correct = sum([score > threshold for score in aa_scores_correct])
     n_aa_predicted = sum([score > threshold for score in aa_scores_all])
     return n_aa_correct / n_aa_predicted, n_aa_correct / n_aa_total
+
+
+def _get_true_labels(mgf_filename: str):
+    """
+    Extract a list of peptide labels from an annotated .mgf and make it compatible with depthcharge AA vocabulary
+
+    Parameters:
+    -----------
+    mgf_filename : str
+        Path to the .mgf file to extract from.
+
+    Returns:
+    --------
+    mgf_df : pandas DF
+        The true peptide labels with column "true_seq"
+    """
+    seqs = []
+    with mgf.MGF(mgf_filename) as f_in:
```

Comment:
We can generalize this to a list of mgf files rather than a single file at a time